nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/io/DocumentModelCSVWriterTest.java

```diff
@@ -70,5 +82,8 @@ public void testDefault() throws Exception {
         csv.has("title").isEquals("myDoc");
         csv.has("dc:description").isEquals("There is a , in the description");
         csv.has("dc:contributors").isEquals("John" + LIST_DELIMITER + "Jane");
+        csv.has("dc:nature").isEquals("article");
+        csv.has("dc:nature[label]").isEquals("Article EN");
+        csv.has("dc:subjects[label]").isEquals("Art" + LIST_DELIMITER + NULL_PROPERTY_LABEL);
```

Comment:
In tests in general it's better to not use the actual values of the implementation. This is so that we detect regressions. Imagine that some changes NULL_PROPERTY_LABEL by mistake. The test uses it too so wouldn't detect the regression.